components/Reminder.js

```diff
@@ -0,0 +1,193 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+const StyledSection = styled.section`
+  position: relative;
+  position: absolute;
+  left: 0;
+  right: 0;
+  margin-left: auto;
+  margin-right: auto;
+  z-index: 999;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 30px 5%;
+  padding: 0 5%;
+  border-radius: 30px;
+  background: var(--light-green);
+  box-shadow: 15px 15px 30px #bebebe, -15px -15px 30px #ffffff;
+`;
+
+const StyledList = styled.ul`
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  justify-content: center;
+  padding: 0;
+`;
+
+const StyledItem = styled.li`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0;
+  list-style: none;
+  border-spacing: 10px;
+`;
+const StyledPlant = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  color: grey;
+  padding: 10px;
+`;
+const StyledName = styled.p`
+  font-weight: bold;
+  color: var(--primary-color);
+  background-color: white;
+  border-radius: 1rem;
+  padding: 0.4rem;
+`;
+const StyledParagraph = styled.p`
+  font-weight: bold;
+  background-color: white;
+  color: var(--dark-yellowish);
+  border-radius: 1rem;
+  padding: 0.4rem;
+`;
+
+const StyledImg = styled.img`
```

Comment:
I recommend using the Image component from Next.js:

```suggestion
const StyledImg = styled(Image)`
```

You can extend its styling as shown above